### PR TITLE
build: start using Vite

### PR DESCRIPTION
**Pull request by eoaksnes**
## What does this pull request change?

* Change to Vite 🔥 
* Adde `on release` Github action workflow
  * Publish Docker image of the example app (needed for Radix) 
  * Deploy to dev environment running in Radix  (currently the radixconfig.yaml file is located in another repository)
* Updated `reset-app.sh` script to handle remote DMSS instances 🚀 
* Added authentication to the application (disabled in Playwright tests) 🔒 
* Renamed `reset-all.sh` -> `reset-app.sh`

> **NOTE**: Need to delete the dmss-data folder, since have changed the DB password!

### CODE REVIEW

**Comment by henrikbossart** [comment]
Seems to work fine in local development. No clue if the `on release` stuff works though. Can I test that somehow, or maybe we'll just find out sooner or later?

Sidenote: I've gotten a bunch of ESlint issues after installing these packages, do you? 🤔 seems to be missing some packages..

**Review by collinlokken — APPROVED** [review]
Looks good overall! A suggestion is to have the JobControlButton change in a different PR

File: example/src/plugins/job-ui-single/pages/JobControlButton.tsx
[the remark this answers is not in the document]
Comment (henrikbossart) [reply example/src/plugins/job-ui-single/pages/JobControlButton.tsx:1]: Its not, buy Vite is a lot stricter than CRA on TS, so it was something that was not picked up by the CI earlier (which is weird).

File: example/vite-env.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
[the remark this answers is not in the document]
Comment (henrikbossart) [reply example/vite-env.d.ts:1]: Its not commented out per-say, its just a vite declare-thingy

**Review by henrikbossart — APPROVED** [review]

**Pull request merged**